### DISCUSSION TITLE
Atualização do Schema XML do Ficheiro de Auditoria Standard para Fins Fiscais (SAF-T) para Angola

### DISCUSSION THREAD

**Discussion by cryptolopes** [assoft-portugal/SAF-T-AO#240] [Announcements]
# Novas funcionalidades

É com grande entusiasmo que anunciamos a maior atualização do repositório [**SAF-T (AO**)](https://github.com/assoft-portugal/SAF-T-AO) desde 2021. As melhorias, agora disponíveis no branch principal ([**master**](https://github.com/assoft-portugal/SAF-T-AO/tree/master)), são fruto de anos de dedicação e colaboração com os nossos Associados e os principais stakeholders envolvidos no processo de implementação do SAF-T em Angola. Este marco representa um avanço significativo, refletindo o nosso compromisso com a melhoria contínua do processo de implementação do SAF-T em Angola.

![AttachedImage](https://github.com/user-attachments/assets/e8f9603d-e1a4-404a-bea5-b651b40ad9f8)

O próximo passo é a disponibilização de uma nova release, que simplificará o download do schema. No entanto, é crucial que esta última versão seja amplamente testada pela comunidade. Queremos assegurar que tudo está em ordem e que a sua utilização é segura. Contamos com a vossa colaboração para garantir a máxima qualidade e segurança.

## Principais novidades

- **Documentação**: foram adicionadas notas explicativas em diversos elementos do SAF-T (AO) diretamente no schema.
- **Correções e melhorias:** WorkingDocuments, TaxBase e mais recentemente foi introduzida a opção INT no elemento TaxCode.
- **Wiki**: atualizamos também a homepage da [**Wiki**](https://github.com/assoft-portugal/SAF-T-AO/wiki) com alguns detalhes sobre o SAF-T (AO), de forma a facilitar o onboarding de novos membros.

## Próximos passos

- Nova release para download
- Melhoria da documentação técnica
 

# Testes de conformidade

Estamos a desenvolver um documento detalhado que visa descrever os testes de conformidade necessários para a validação das aplicações de software de faturação. Este documento será baseado na documentação fornecida pela AGT - Administração Geral Tributária, e incluirá exemplos práticos e dicas para otimizar o software, assegurando a conformidade de todos os processos. A conclusão desta documentação ainda não data fixa, mas esperamos disponibilizar uma primeira versão em breve.

# Envolva-se

Participe no grupo de trabalho wg-saft-ao e acompanhe as reuniões quinzenais. Assegure a conformidade do seu software através do acompanhamento contínuo deste projeto. A sua participação é fundamental para o sucesso coletivo!

## Participe através dos seguintes canais

- [**Microsoft Teams**](https://protect.checkpoint.com/v2/___https:/teams.microsoft.com/l/meetup-join/19%3a1b4b49dbf83d4f42b205f7df89e85867%40thread.skype/1668019808208?context=%7b%22Tid%22%3a%22ec237b6a-26a1-4fb6-849a-0564a8196832%22%2c%22Oid%22%3a%22161a2b31-3bd2-4b00-a6e7-f09cdbd5effd%22%7d___.YzJlOm1pbmlzdHJ5b2ZmaW5hbmNlYW5nb2xhOmM6bzo1MTY1OTRiOGVhNmQwZTBkMTBkNTc3Mjc5NjU5MTkxMzo2OjQ1NTk6ZTE4ODI5ZmE3NjY4ZGFjOWJlYTY2YmYwYjMwNTc2NGUxYzIxNjgxODU5YTQ5YzkzNTg1YThjNDhhNzc5MTUxOTpoOlQ) (de 15 em 15 dias)
- [**Telegram**](https://t.me/saftao) (SAF-T AO)
- [**Github**](https://github.com/assoft-portugal/SAF-T-AO)

**Comment by AfonsoMenata** [comment]
Como pego o link da reunião??
